src/error.rs

```diff
@@ -31,5 +31,25 @@ error_chain! {
             description("functionality is not implemented for this platform")
             display("{} is not implemented for the {} platform", functionality, platform)
         }
+
+        MissingFrontStart {
+            description("Malformed jekyll document, missing frontmatter start")
+            display("Malformed jekyll document, missing frontmatter start")
+        }
+
+        MissingFrontmatter {
+            description("Malformed jekyll document, missing frontmatter")
```

Comment:
If these errors are specific to Jekyll, they should probably have that in their name.

Reply:
The description is clumsy, I think this error can happen for cobalt's document too. May I change the description to make it more generic?

Reply:
In #203, we decided to support missing front matters, so the error only applies to Jekyll.

Reply:
Ok, I'll move it to jk_document then. By curiosity: what's the title if no front is present?

Reply:
We turn the slug into a title
https://github.com/cobalt-org/cobalt.rs/blob/master/src/frontmatter.rs#L283

Otherwise we fallback to a default
https://github.com/cobalt-org/cobalt.rs/blob/master/src/frontmatter.rs#L321

Reply:
Are you sure about the default fallback? `ok_or_else` returns a `Result` and with the `?` if no title, we generate an `Error` with the message "No title", aren't we?